backend/app.ts

```diff
@@ -0,0 +1,76 @@
+import * as dotenv from 'dotenv';
+import path from 'path';
+dotenv.config({ path: path.resolve(__dirname, "../.env") });
+import express from 'express';
+import cors from 'cors';
+import morgan from 'morgan';
+import session from 'express-session';
+import passport from 'passport';
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
+import googleOAuthRouter from './routes/google-oauth';
+import testRouter from './routes/test-route'
+import prisma from './utils/prismaClient';
+
+const app = express();
+
+dotenv.config();
+app.use(morgan("dev"));
+app.use(cors());
+app.use(express.json());
+app.use(session({
+    secret: "secretcode",
```

Comment:
We want to make sure this is secret, so technically, this should be pulled in from an `ENV var